### DISCUSSION TITLE
[New Feature] Harvest animations for (SHP) harvesters

### DISCUSSION THREAD

**Discussion by Bittah** [Vinifera-Developers/Vinifera#1466] [Feature Requests]
### Description:
In TD and RA1 harvesters animate when they harvest, but TS lacks this feature.

### Possible Implementation:
Either just make use of `FiringFrames=` and `StartFiringFrame=` in art.ini and display the firing animation whenever the harvester harvests a patch or add separate keys like `HarvestFrames=` and `StartHarvestFrame=`.

### Additional Files:

https://user-images.githubusercontent.com/7165430/122831927-e15bc480-d2ea-11eb-97db-7b421f3dbf88.mp4

**Comment by JoyfulShush** [comment]
I think this one is important for making non-vanilla harvesters look good when harvesting. In DTA, for example, it is impossible to see whether harvesters can harvest or not. Allowing to modify what frames or animations play when they harvest would improve visibility significantly.
